docs/mobile-apps/automated-testing/appium/appium-flutter-integration-driver.md

```diff
@@ -0,0 +1,678 @@
+---
+id: appium-flutter-integration-driver
+title: Appium Flutter Integration Driver Testing on Sauce Labs
+sidebar_label: Appium Flutter Integration Testing
+description: Learn how to test Flutter apps with the Appium Flutter Integration Driver.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Sauce Labs supports testing Flutter apps on Android and iOS virtual and real devices with Appium by utilizing the [appium-flutter-integration-driver](https://github.com/AppiumTestDistribution/appium-flutter-integration-driver). 
+The process to test Flutter apps with Appium involves an additional and crucial step: [Prepare the app with Flutter Integration Server](#prepare-the-app-with-flutter-integration-server). 
+This step includes instructions on how to prepare and build your app for both Android and iOS platforms. After preparing your app, 
+you can [upload](#uploading-your-flutter-app-to-sauce-labs) it to Sauce Labs, [Configure your Appium capabilities](#configuring-your-appium-capabilities), and run your tests.
+
+
+## Native Flutter Integration Driver vs Appium Flutter Integration Driver
+
+| Use Cases                                                                                                                           | Native Flutter Driver | Appium Flutter Integration Driver |
+| ----------------------------------------------------------------------------------------------------------------------------------- | --------------------- | --------------------------------- |
+| Writing tests in languages other than Dart                                                                                          | ❌                    | ✔️                                |
+| Running integration tests for Flutter apps with embedded webview or native view, or existing native apps with embedded Flutter view | ❌                    | ✔️                                |
+| Running tests on multiple devices simultaneously                                                                                    | ❌                    | ✔️                                |
+| Running integration tests on device farms that offer Appium support                                                                 | ❌                    | ✔️                                |
+| App interactions beyond Flutter’s contextuality (e.g., sending an OTP from a message application)                                   | ❌                    | ✔️                                |
+
+## Differences from Appium Flutter Driver
+
+The current Appium Flutter Driver is built on top of the `flutter_test` SDK, which is deprecated. The potential deprecation ([Expand deprecation policy to package:flutter_driver](https://github.com/flutter/flutter/issues/139249)) means this driver may not work with future Flutter updates. It also does not handle all cases, such as permission dialog handling.
```

Comment:
This bit `Expand deprecation policy to package:flutter_driver` looks strange. 

<img width="977" alt="image" src="https://github.com/saucelabs/sauce-docs/assets/5992658/a85a139a-ff1c-43f8-a583-f472c5383dad">

Reply:
Did you mean just to link the word `deprecation`?

Reply:
no. that is correct. 
Source of this is here : https://github.com/AppiumTestDistribution/appium-flutter-integration-driver